docker-compose/tail-sampling/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+services:
+  jaeger:
+    networks:
+      backend:
+    image: jaegertracing/jaeger:${JAEGER_IMAGE_TAG:-latest}
+    volumes:
+      - "./jaeger-v2-config.yml:/etc/jaeger/config.yml"
+    command: ["--config", "/etc/jaeger/config.yml"]
+    ports:
+      - "16686:16686"
+
+  otel_collector:
```

Comment:
what's the purpose of otel collector? To illustrate the use of loadbalancer exporter? It looks like you are actually bypassing it since tracegen sends data to `jaeger:4318`

Reply:
Yes - I was trying to illustrate the use of the loadbalancer exporter. Is there a way to have tracegen send data to the load balancer? Or do we want to take out the loadbalancer for the example.

Reply:
you can, you just need to provide correct URL to tracegen

Reply:
@yurishkuro I tried changing my configuration to https://gist.github.com/mahadzaryab1/0b8ccc194421e00cd2e6dce6f450c424 but am getting the following error. 
```
otel_collector-1  | 2024-08-31T05:01:44.318Z    info    exporterhelper/retry_sender.go:169      Exporting failed. Will retry the request after interval.     {"kind": "exporter", "data_type": "traces", "name": "loadbalancing", "error": "rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup jaeger on 127.0.0.11:53: no such host\"", "interval": "3.852639674s"}
```

Do you know what I'm missing here?

Reply:
probably because of how your loadbalancing exporter is configured:
```
    resolver:
      static:
        hostnames:
        - 0.0.0.0:4317
```
Probably should be `jaeger:4317` (and make `jaeger` export 4317 port), right now it only exposes "16686:16686"

Reply:
@yurishkuro do I also need to change `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` to be `http://otel_collector:4318/v1/traces`? I tried this along with exposing port 4317/4318 and get the following error now
```
otel_collector-1  | 2024-08-31T05:47:57.011Z    info    exporterhelper/retry_sender.go:118      Exporting failed. Will retry the request after interval.     {"kind": "exporter", "data_type": "traces", "name": "loadbalancing", "error": "rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 172.18.0.2:4317: connect: connection refused\"", "interval": "7.19103943s"}
```

Reply:
* you need to make sure collector and jaeger are listening on 0.0.0.0 IP
* you only need 4318 port (http) open on the collector, and only 4317 (grpc) on jaeger

Reply:
@yurishkuro thanks! this works with the load balancer now. i'll document it in the README.